stl/inc/bit

```diff
@@ -0,0 +1,33 @@
+// bit standard header

```

Comment:
```suggestion
// bit standard header (core)
```

A subset of our headers, like `<type_traits>`, are usable without any runtime support machinery. We indicate this by saying `(core)` in their introductory comment. This is true for `<bit>` as well (even after the power-of-2 etc. functions are added, they'll be powered by compiler builtins, nothing separately compiled).